### DISCUSSION TITLE
Multiple ID/relay servers (automatically select the relay server closest to you)

### DISCUSSION THREAD

**Discussion by Accademia** [rustdesk/rustdesk#9085] [Q&A] [answered]
Why can't this feature be provided in the client? Specifically, if I set up multiple ID servers and relay servers, I would like to be able to configure them all in my client simultaneously. This way, if one server fails (especially if it is blocked by the Great Firewall of China), the client can automatically connect to another set of servers, thus avoiding disruptions in communication between multiple clients.

Please note, what I am looking for is the ability to configure multiple servers on the client side, not just multiple relay servers on the server side ( https://rustdesk.com/docs/en/self-host/rustdesk-server-pro/relay ). The latter would still result in the inability to log in if the ID server is blocked by the Great Firewall of China.

**Comment by Accademia** [comment]
Please note, what I am looking for is the ability to configure multiple servers on the client side, not just multiple relay servers on the server side ( https://rustdesk.com/docs/en/self-host/rustdesk-server-pro/relay ). The latter would still result in the inability to log in if the ID server is blocked by the Great Firewall of China.

**Comment by rustdesk** [comment, marked as answer]
https://github.com/rustdesk/rustdesk/discussions/1258
